### PR TITLE
Use is-buffer instead of using Buffer object

### PR DESCRIPTION
**Pull request by phieronymus**
Remove unnecessary dependency on the node.js Buffer. This is now using a lighter object that does not need polyfills

### CODE REVIEW

File: src/utils/object.ts [hunk 1]
```diff
@@ -24,11 +25,11 @@ export const transformKeys = (
     transformer: Transformer,
     ignoreKeys: string[] = []
 ): any => {
-    const transformArray = (arr: unknown[], transformer: Transformer): any =>
+    const transformArray = (arr: unknown[]): any =>
         arr.map((item) => (isObject(item) ? transformKeys(item, transformer, ignoreKeys) : item));
 
     if (Array.isArray(obj)) {
-        return transformArray(obj, transformer);
+        return transformArray(obj);
```
Comment (phieronymus) [inline src/utils/object.ts:32]: Yeah... This is not really related but there was a shadow variable... Better fix that now 🙂

**Review by laxa88 — APPROVED** [review]
